ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -266,7 +266,6 @@
 				"${BUILT_PRODUCTS_DIR}/image_picker/image_picker.framework",
 				"${BUILT_PRODUCTS_DIR}/leveldb-library/leveldb.framework",
 				"${BUILT_PRODUCTS_DIR}/nanopb/nanopb.framework",
-				"${BUILT_PRODUCTS_DIR}/package_info/package_info.framework",
```

Comment:
What is the purpose of this change?

Reply:
Some generated file by Xcode, I ran the app on iOS and this was an automatic change.